tests/setup_test.go

```diff
@@ -2252,3 +2254,68 @@ func TestRefreshRemoteRepositoryInfo(t *testing.T) {
 		t.Fatalf("projects mismatch (-expected +got):\n%s", diff)
 	}
 }
+
+func TestAddUpdateOrgUserMembers(t *testing.T) {
+	dir, err := ioutil.TempDir("", "agola")
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	defer os.RemoveAll(dir)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	_, c := setup(ctx, t, dir, false)
+
+	gwClient := gwclient.NewClient(c.Gateway.APIExposedURL, c.Gateway.AdminToken)
+
+	_, _, err = gwClient.CreateUser(ctx, &gwapitypes.CreateUserRequest{UserName: agolaUser01})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	_, _, err = gwClient.CreateOrg(ctx, &gwapitypes.CreateOrgRequest{Name: agolaOrg01, Visibility: gwapitypes.VisibilityPublic})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	//test add org member role member
+	orgMember, _, err := gwClient.AddOrgMember(ctx, agolaOrg01, agolaUser01, gwapitypes.MemberRoleMember)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	orgMembers, _, err := gwClient.GetOrgMembers(ctx, agolaOrg01)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	if orgMembers == nil {
+		t.Fatal("unexpected members nil")
+	}
+	if len(orgMembers.Members) != 1 {
+		t.Fatalf("expected Members len 1, got: %d", len(orgMembers.Members))
+	}
+	if diff := cmp.Diff(*orgMembers.Members[0], orgMember.OrgMemberResponse); diff != "" {
+		t.Fatalf("org member mismatch (-expected +got):\n%s", diff)
+	}
+
+	//test update org member role owner
+	orgMember, _, err = gwClient.AddOrgMember(ctx, agolaOrg01, agolaUser01, gwapitypes.MemberRoleOwner)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+
+	orgMembers, _, err = gwClient.GetOrgMembers(ctx, agolaOrg01)
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	if orgMembers == nil {
+		t.Fatal("unexpected members nil")
+	}
+	if len(orgMembers.Members) != 1 {
+		t.Fatalf("expected Members len 1, got: %d", len(orgMembers.Members))
+	}
+	if diff := cmp.Diff(*orgMembers.Members[0], orgMember.OrgMemberResponse); diff != "" {
```

Comment:
You should explicitly define the expected response and not rely on a server response.